From f313cf09f54676c78df65a4e3a6399586be9f9bb Mon Sep 17 00:00:00 2001
From: Ben Greear <greearb@candelatech.com>
Date: Wed, 22 Jul 2020 16:34:16 -0700
Subject: [PATCH] backports-ct:  Align ath9k header with 5.7.10-ct backports.

I removed a bunch of conflicting ath9k patches since I don't plan
to use ath9k on openwrt anytime soon.

This is not something that should go upstream.

Signed-off-by: Ben Greear <greearb@candelatech.com>
---
 .../linux/generic/files/include/linux/ath9k_platform.h   | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/target/linux/generic/files/include/linux/ath9k_platform.h b/target/linux/generic/files/include/linux/ath9k_platform.h
index f1f2ad419c..76860a461e 100644
--- a/target/linux/generic/files/include/linux/ath9k_platform.h
+++ b/target/linux/generic/files/include/linux/ath9k_platform.h
@@ -46,15 +46,6 @@ struct ath9k_platform_data {
 	int (*external_reset)(void);
 
 	bool use_eeprom;
-
-	int num_leds;
-	const struct gpio_led *leds;
-
-	unsigned num_btns;
-	const struct gpio_keys_button *btns;
-	unsigned btn_poll_interval;
-
-	bool ubnt_hsr;
 };
 
 #endif /* _LINUX_ATH9K_PLATFORM_H */
-- 
2.20.1

